modules/greatest/v1.5.0/MODULE.bazel

```diff
@@ -0,0 +1,7 @@
+module(
+    name = "greatest",
+    version = "v1.5.0",
```

Comment:
The convention in BCR is to have "1.5.0" than "v1.5.0". Do you think if you could change this?

Reply:
Thanks! You have to also change the directory name.